solutions/swb-reference/src/constants.ts

```diff
@@ -207,13 +207,18 @@ const endPointPrefix: string = 'ENDPOINT';
 const authorizationGroupPrefix: string = 'GROUP';
 
 const enum SwbAuthZSubject {
+  SWB_AWS_ACCOUNT = 'SWB_AWS_ACCOUNT',
+  SWB_AWS_ACCOUNT_TEMPLATE_URL = 'SWB_AWS_ACCOUNT_TEMPLATE_URL',
+  SWB_COST_CENTER = 'SWB_COST_CENTER',
   SWB_DATASET = 'SWB_DATASET',
   SWB_DATASET_ACCESS_LEVEL = 'SWB_DATASET_ACCESS_LEVEL',
   SWB_ENVIRONMENT = 'SWB_ENVIRONMENT',
   SWB_ENVIRONMENT_CONNECTION = 'SWB_ENVIRONMENT_CONNECTION',
   SWB_ENVIRONMENT_TYPE = 'SWB_ENVIRONMENT_TYPE',
   SWB_ETC = 'SWB_ETC',
   SWB_PROJECT = 'SWB_PROJECT',
+  SWB_PROJECT_USER_ASSOCIATION = 'SWB_PROJECT_USER_ASSOCIATION',
+  SWB_ROLE = 'SWB_ROLE',
```

Comment:
this isn't being used

Reply:
SWB_PROJECT_USER_ASSOCIATION is used by `/projects/:projectId/users/:userId`
Correct that SWB_ROLE isn't used, that will be deleted.